### ISSUE TITLE
Qalculate GTK doesn't always copy the results to clipboard

### ISSUE THREAD

**Issue by arrufat** [Qalculate/qalculate-gtk#348]
I've noticed that, unless I press enter after a calculation and the results gets added to the history, the copy result functionality does not work.

I came across this issue because I set up Qalculate! not to add the results to the history unless I press enter.
If I change the setting to add the results to the history automatically after some delay, then the copy to clipboard functionality works as expected (even without pressing enter).

So, to summarize, we can't copy results unless they are added to the history.
To copy results, I tried:
- clicking on the results (a pop-up shows up telling that the result has been copied)
- through the edit menu.

**Comment by arrufat** [comment]
Thank you!
